src/kOS.Safe/Compilation/KS/kRISC.tpg

```diff
@@ -95,8 +95,9 @@ COMMENTLINE  -> @"//[^\n]*\n?";
 // Rules
 // ===================================================
 Start -> (instruction)* EOF;
-instruction_block -> CURLYOPEN instruction* CURLYCLOSE EOI?;
-instruction -> set_stmt | 
+instruction_block -> CURLYOPEN instruction* CURLYCLOSE;
+instruction -> empty_stmt |
+               set_stmt | 
```

Comment:
These changes were because this syntax didn't work before:
```
set foo to {return 0.}.
```
and instead required two dots, which looked kinda dumb:
```
set foo to {return 0.}..
```
It required 2 dots because the old version of this rule insisted on taking the first dot as being part of the `instruction_block` instead of as the statement terminator, thus leaving the statement unterminated.
This change makes it cleaner by saying that the optional dot after the braces isn't in fact part of the instruction block at all, but is just an empty statement, which is now legal.